src/FAPOFX_echo.c

```diff
@@ -46,9 +90,9 @@ const FAudioGUID FAPOFX_CLSID_FXEcho =
 	}
 };
 
-static FAPORegistrationProperties FXEchoProperties =
+static FAPORegistrationProperties FXEchoProperties_LEGACY =
 {
-	/* .clsid = */ {0},
+	/* .clsid = */ FAPOFX_CLSID_FXEcho_LEGACY,
```

Comment:
I’d have to double check tonight (currently away from a PC), but I believe MinGW complains about this clsid initialization. I totally forget why! For reference I’m building with mingw-w64 from Fedora 28.

Everything else lgtm!

Reply:
Ah I was wondering why that memcpy was there, it seemed obviously wacky. I can change it back.